### PR TITLE
add brand text free 

### LINKED ISSUE #228: add brand input

### PR DESCRIPTION
**Pull request by amanshawar**
add brand text free box and change it in everyplace should be modifit…ed fix #228

### CODE REVIEW

File: client/src/components/Shared/Form/form.style.js [hunk 1]
```diff
@@ -85,6 +85,18 @@ export const StyledInput = styled.input`
   }
 `;
 
+export const StyledInputBrand = styled.input`
+  background-color: #fbfbfb;
+  font-size: 20px;
```
Comment (jema28) [inline client/src/components/Shared/Form/form.style.js:90]: The font size is currently too big, let's reduce this to 16px to match the other inputs? Unrelated to this pull request but the price field should be changed also.

Would be great to match the free text colour of select ones to make all fields identical 😸 

Screenshot 2019-04-23 at 17 56 56

File: client/src/components/Shared/Form/index.js [hunk 2]
```diff
@@ -182,20 +183,13 @@ class Form extends Component {
...
-                isRtl={false}
-                isSearchable
-                name="Brand"
-                options={brands}
-                onChange={handleChange}
-                styles={this.errorStyle(isErrorBrand)}
-              />
-            </SelectStyle>
+            <StyledInputBrand
+              onChange={toggleOpen}
```
Comment (jema28) [inline client/src/components/Shared/Form/index.js:187]: I don't think we need this toggleOpen anymore.

**Review by jema28 — CHANGES_REQUESTED** [review]
Great job! Just a few stylistic comments 😸

File: client/src/components/Shared/Form/form.style.js (hunk 1 above)
Comment (amanshawar) [reply client/src/components/Shared/Form/form.style.js:90]: it's 16
Would be great to match the free text colour of select ones to make all fields identical smile_cat

 whats this

File: client/src/components/Shared/Form/index.js (hunk 2 above)
Comment (amanshawar) [reply client/src/components/Shared/Form/index.js:187]: need

**Pull request merged**